src/styles/clock.scss

```diff
@@ -0,0 +1,67 @@
+.clock {
+  position: relative;
+  width: 300px;
+  height: 300px;
+  border: 3px solid $minute-color;
+  border-radius: 50%;
+
+  &::before {
+    content: "";
+    position: absolute;
+    display: block;
+    width: 20px;
+    height: 20px;
+    background-color: $minute-color;
+    border-radius: 50%;
+    top: 47%;
+    left: 47%;
+    z-index: 1;
+  }
+
+  &__minute {
+    z-index: 1;
+    position: absolute;
+    display: block;
+    width: 10px;
```

Comment:
If you repeat this value in many parts of the code (10px), you should create a variable.